### PR TITLE
add openshift-monitoring configuration

### PR DESCRIPTION
**Pull request by droslean**

### CODE REVIEW

**Comment by hongkailiu** [comment]
I would suggest:
1. back up the current rule
```
$ oc get prometheusrule -n openshift-monitoring --as system:admin > prometheus-k8s-rules_prometheusrule.yaml
```
2. edit the rule to add your alerts
3. include the update in our CI
```
$ oc apply -f prometheus-k8s-rules_prometheusrule.yaml
```

**Comment by droslean** [comment]
/hold

I am not sure if just updating the configMap is enough. Checking...

File: cluster/ci/openshift-monitoring/prometheus-k8s-rules_prometheusrule.yaml [hunk 1]
```diff
@@ -0,0 +1,1454 @@
...
+        annotations:
+          description: '{{$labels.namespace}}/{{$labels.pod}} has many samples rejected
+            due to duplicate timestamps but different values'
+          summary: Prometheus has many samples rejected
+        expr: |
+          increase(prometheus_target_scrapes_sample_duplicate_timestamp_total{job="prometheus-k8s"}[5m]) > 0
+        for: 10m
+        labels:
+          severity: warning
+    - name: prow.rules
```
Comment (droslean) [inline cluster/ci/openshift-monitoring/prometheus-k8s-rules_prometheusrule.yaml]: This is the group added for prow. Before that line, all the group are already exist.

**Comment by droslean** [comment]
/hold cancel

Update the prometheus CR instead.

**Pull request merged**